package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "leftovers-fe",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint . --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@material-tailwind/react": "2.1.9",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "sass": "1.77.8"
+  },
+  "devDependencies": {
+    "@eslint/compat": "1.1.1",
+    "@types/react": "18.3.3",
+    "@types/react-dom": "18.3.0",
+    "@typescript-eslint/eslint-plugin": "7.16.1",
+    "@typescript-eslint/parser": "7.16.1",
+    "@vitejs/plugin-react-swc": "3.7.0",
+    "autoprefixer": "10.4.19",
+    "cssnano": "7.0.4",
+    "eslint": "9.7.0",
+    "eslint-plugin-react-hooks": "4.6.2",
+    "eslint-plugin-react-refresh": "0.4.8",
+    "postcss": "8.4.39",
+    "prettier": "3.3.3",
+    "prettier-plugin-tailwindcss": "0.6.5",
+    "tailwindcss": "3.4.6",
+    "typescript": "5.5.3",
+    "vite": "5.3.4"
+  },
+  "overrides": {
+    "eslint": "9.7.0"
+  }
```

Comment:
just leaving a note that it's kind of risky, but we have already discussed it over slack 👍🏻